crt_portal/cts_forms/model_variables.py

```diff
@@ -157,6 +157,21 @@
     ('VOT', 'Voting'),
 )
 
+# CRT view global section filter
+SECTION_CHOICES_WITHOUT_LABELS = (
```

Comment:
We should define this as a function of `SECTION_CHOICES` so that we only need to update them in a single place in the future.

Reply:
@Jkrzy do you mean define a python function to create a tuple of `SECTION_CHOICES` with no labels. My question is where would this function reside. I am thinking in ProfileForm but I would love some feedback.

Reply:
I'd recommend using a list comprehension to set the value of `SECTION_CHOICES_WITHOUT_LABELS` here